### DISCUSSION TITLE
Nautical Mile measurements

### DISCUSSION THREAD

**Discussion by Weirwindle** [MoffKalast/Sailwind-Map#14] [General]
May I ask how nautical mile distance measurements were pulled from game?
You may know about the Chip Log problem where people taking the measured speed in knots and dividing by 1.6 to get, what they think to be, their actual speed. I would guess that this is from your Sailwind Interactive map giving measurements as fact.
Your map is an incredible tool BTW. I have been using it over the last week to calculate dead reckoning plots by only using the chronometer, chip log, and compass. Using the Dhow on calm seas I am able to complete the course from Gold Rock City to Oasis with under a 4nm / 1 hour error. For this I must either do the chip log speed/1.6 conversion to match up with your map, or I can use the tools in game and their given measurements, and therefore multiply your map measurements by 1.6 to get an agreeing answer.
I have been trying to find the route of the discrepency.
Shouldn't 1 nautical mile equate to 1/60th of a degree.
The distance from Gold Rock City to Dragon Cliffs measures 870nm. There are ~9.8 degrees of longtitude (588 arcminutes/nautical miles) given the definition. This makes the discrepency doubly worse.
Given there are 90 arcminutes between Gold Rock City and Oasis, and it takes 52 hours for the trip, that would be an average speed of 1.7kts. Far off of the ~4.5kts the chip log gives or the 2.8kts the 4.5/1.6 gives.
I also believe that an ancient dhow should be able to sail 4-6 kts as shown here https://penelope.uchicago.edu/Thayer/E/Journals/TAPA/82/Speed_under_Sail_of_Ancient_Ships*.html
rather than the 2.5-3.75 that the /1.6 conversion gives, or the 1.5-2.3 that a 1 arcminute nautical mile gives.
Thank you for your time,
Weirwindle

**Comment by Weirwindle** [comment]
I stumbled across this **https://steamcommunity.com/app/1764530/discussions/0/4731597528360202807/?l=finnish**

yeah uh... there's some trouble regarding the translation from knots to "miles".

I *think* this is how it works:
1 "in-game mile" is internally known as 1 "km", and is actually equal to 100 meters if scaled against the player.
1 "knot" is 1.852 "km"/h.
1 in-game day is 50 minutes IRL

so for example, I covered 870 "mile" in 6.5 days at 7-9 knots
9 nmi per hour x 5/6 = 7.5 nmi per 50 minutes, i.e. per in game day
7.5 x 6.5 days = 48.75 "actual" nmi covered in 6.5 in-game days
48.75 x 1.852 = 90.285 "actual" km covered in 6.5 in-game days
1 "in-game km" is 100m instead of 1000 IRL so 90.285 actual km x 10 = 902.85 in-game km = 902.85 in-game mile.

Considering I will have some deviations, plus I'm not 9 knots all the way, I'd call that fair.

**Comment by MoffKalast** [comment]
Hey, glad you like it. I've never really looked into it past the 1 deg = 90 NM conversion value that the community claims to be correct. This is probably more of a discussion for the discord channel, a few people there have got it all figured out :smile: 

Anyhow, maybe it would be neat if we had a toggle to also display the distance in meters, calculated in actual scale of the player. It would certainly be even more confusing though.
